_posts/2020-11-23-micro-grants-in-2021-ols-3-ols-4.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: OLS Community support micro grants
+authors:
+- bebatut
+- malvikasharan
+- yochannah
+image: https://images.unsplash.com/photo-1579621970563-ebec7560ff3e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1951&q=80
+photos:
+  name: Photo by Micheile Henderson on Unsplash
+  url: https://unsplash.com/@micheile
+---
+
+Open Life Science has recently been lucky enough to be awarded two small grants - one from the EOSC-Life project, which will partially support OLS-3, and two more small grants which will partially support OLS-3 and 4. In all cases, the funds awarded are less than the full cost of a cohort if we were to pay for all the things we would like to pay for, including mentor and expert speaker time, but nevertheless - it is a helpful start, and one we are privileged to have.
```

Comment:
```suggestion
Open Life Science has recently been awarded three grants - one from the EOSC-Life project, which will partially support OLS-3, and two more small grants which will partially support OLS-3 and 4. The full cost of these two cohorts are higher than the current budget if we were to pay for all the things we would like to pay for, including mentor and expert speaker time, but nevertheless - it is a significant start and one we are privileged to have.
```

Reply:
I think we don't need to add this here "The full cost of these two cohorts are higher than the current budget if we were to pay for all the things we would like to pay for, including mentor and expert speaker time, but nevertheless - it is a significant start and one we are privileged to have."